tests/page-objects/dashboard/DashboardPage.ts

```diff
@@ -85,6 +85,8 @@ class DashboardPage extends BasePage {
         acceptableSelectBtn: 'select-acceptable-button',
         dialogApplyBtn: 'dialog-apply-button',
         dialogCancelBtn: 'dialog-cancel-button',
+        chartGridItemContainer: `${slct(COMMON_DASH_SELECTORS.DASH_GRID_ITEM)} .chartkit`,
+        dashPluginWidgetBody: slct('chart-widget'),
```

Comment:
I'm not sure it's a good idea to change api (adding `slct()`) for `selectors` object only for two new values. This will be confusing for future usage

Reply:
It's just selectors. It's preferable to put selectors in one location rather than having many files for constants.

Reply:
The `slct()` creates a wrapper to use the `data-qa` attribute. As a result you get just a selector.